### ISSUE TITLE
Developers doc is unaccessible

### ISSUE THREAD

**Issue by Atlanta** [crestic-urca/remotelabz-docs#3]
In GitLab by @Atlanta on Jun 27, 2020, 12:22

https://docs.remotelabz.com/developpers/getting-started/vscode.md returns a 404 error.

**Comment by Atlanta** [comment]
In GitLab by @Atlanta on Jun 27, 2020, 12:22

created merge request !1 to address this issue

**Comment by Atlanta** [comment]
In GitLab by @Atlanta on Jun 27, 2020, 12:22

mentioned in merge request !1

**Comment by Atlanta** [comment]
In GitLab by @Atlanta on Jun 27, 2020, 12:37

Path is wrongly configured in [mkdocs.yml:27](mkdocs.yml).

```yaml
    - 'For Developpers':
        - 'Getting started':
            - 'Visual Studio Code': developers/getting-started/vscode.md
            # there is no folder named getting-started in "developers"
```

**Comment by Atlanta** [comment]
In GitLab by @Atlanta on Jun 27, 2020, 12:38

mentioned in commit 900c8b7da55d8bca6a11315665905dc8f37a4999

**Comment by Atlanta** [comment]
In GitLab by @Atlanta on Jun 27, 2020, 12:38

assigned to @Atlanta

**Comment by Atlanta** [comment]
In GitLab by @Atlanta on Jun 27, 2020, 12:39

closed via merge request !1

**Comment by Atlanta** [comment]
In GitLab by @Atlanta on Jun 27, 2020, 12:39

closed via commit 900c8b7da55d8bca6a11315665905dc8f37a4999

**Comment by Atlanta** [comment]
In GitLab by @Atlanta on Jun 27, 2020, 12:39

mentioned in commit 650157b8ebfa7f605b6a3805c3e0aa1550d4af4a
